README.md

```diff
@@ -56,30 +67,28 @@ Also make sure to include the following in order to import the compiled CSS from
 
 **[More info about using USWDS CSS & SCSS](./docs/scss.md)**
 
-## Roadmap
```

Comment:
fyi: I made sure any outstanding roadmap items were reflected in github issues before deleting this section